tests/test_sklearn_histgbdt_converters.py

```diff
@@ -17,9 +17,10 @@ def _run_GB_trees_classifier_converter(self, num_classes, extra_config={}, label
         warnings.filterwarnings("ignore")
         for max_depth in [2, 3, 8, 10, 12, None]:
             model = HistGradientBoostingClassifier(max_iter=10, max_depth=max_depth)
-            X = [[0, 1], [1, 1], [2, 0]]
+            np.random.seed(0)

```

Comment:
I fixed the data because `hist_gbdt` was generating errors quite often.

Reply:
was this because of a SKL problem? or something else?

Reply:
It is something related to `hist_gbdt` not sklearn per me.